### PR TITLE
Fix iframe and add layouts

### PR DESCRIPTION
**Pull request by nanu-c**
This depends on https://github.com/MILA-Wien/collectivo/pull/113
It introduces a layout based system in the router, has a fall back and enables to use the full space for iframes.

### CODE REVIEW

**Comment by nanu-c** [comment]
grafik

File: src/components/SidebarMenuItem.vue [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    :href="item.url"
+    :class="{ 'p-disabled': item.disabled }"
+    target="_blank"
+  >
+    <div class="flex flex-row py-2 px-3 gap-2 items-center">
+      <span class="flex p-menuitem-icon content-center" v-if="item.icon">
+        <i :class="item.icon"></i>
+      </span>
+
+      <span class="pt-1 w-full">{{ item.target }}{{ item.label }}</span>
```
Comment (JoelForamitti) [inline src/components/SidebarMenuItem.vue:59]: I think {{ item.target }} should not be here, right?

image

File: src/components/SidebarMenuItem.vue [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    v-else-if="item.target == 'iframe'"
+    :to="item.to"
+    :class="{ 'p-disabled': item.disabled }"
+  >
+    <div class="flex flex-row py-2 px-3 gap-2 items-center">
+      <span class="flex p-menuitem-icon content-center" v-if="item.icon">
+        <i :class="item.icon"></i>
+      </span>
+
+      <span class="pt-1 w-full">iframe{{ item.label }}</span>
```
Comment (JoelForamitti) [inline src/components/SidebarMenuItem.vue:39]: Same here

**Review by JoelForamitti — CHANGES_REQUESTED** [review]
Looks great!

**Comment by JoelForamitti** [comment]
Integrated into https://github.com/MILA-Wien/collectivo-ux/pull/85

**Pull request closed without merging**